packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -348,6 +408,35 @@ pipeline.addStage({
 });
 ```
 
+##### Lambda deployments
+
+To use CodeDeploy for blue-green Lambda deployments in a Pipeline:
+
+```typescript
+const lambdaCode = lambda.Code.asset('path/to/directory/or/zip/file'); 
+const func = new lambda.Function(lambdaStack, 'Lambda', {
+  code: lambdaCode,
+  handler: 'index.handler',
+  runtime: lambda.Runtime.NodeJS810,
+});
+// used to make sure each CDK synthesis produces a different Version
+const version = func.newVersion();
```

Comment:
This sounds weird and scary?... We can't have "cdk synth" produce a different result each time. Is this just temporary until we have #1400?

Reply:
It's meant to be used in a Pipeline, so I believe that's fine.